store/modules/playback/actions.js

```diff
@@ -193,8 +193,12 @@ export default {
   },
   async validate_barcode({ commit, state, dispatch }, { type, new_value }) {
     const result = TextValidation_plate_barcode.validate(new_value);
-    const is_valid = result == "";
+    let is_valid = result == "";
```

Comment:
could shorten to:
`const is_valid = result == "" && !this.state.flask.barcode_manual_mode`